coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -250,4 +251,58 @@ def extract_documentation_with_markers(content, docstyle_definition):
             marker_dict,
             docstyle_definition)
         if doc:
+            # Instantiate padding
+            top_padding = 0
+            bottom_padding = 0
+            start_index = doc.range.start.line - 2
+            end_index = doc.range.end.line
+            while start_index >= 0 and content[start_index] == '\n':
```

Comment:
You can't assume that users are also using other bears. The bear needs to work independently